File: qlack-be-rules/qlack-be-rules-ui/angular/src/app/dto/working-set-dto.ts
import {BaseDto} from './base-dto';
import {CategoryDto} from './category-dto';
import {WorkingSetVersionDto} from './working-set-version-dto';

export interface WorkingSetDto extends BaseDto {
  name?: string;
  description?: string;
  status?: boolean;
  workingSetVersions?: WorkingSetVersionDto[];
  categories?: CategoryDto[];
}

File: qlack-be-rules/qlack-be-rules-server/src/main/java/com/eurodyn/qlack/rulesdemo/dto/WorkingSetDTO.java
package com.eurodyn.qlack.rulesdemo.dto;

import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;

import java.util.List;

@Data
@Accessors(chain = true)
@EqualsAndHashCode(callSuper = true)
public class WorkingSetDTO extends BaseDTO {

    private String name;

    private String description;

    private boolean status;

    private List<WorkingSetVersionDTO> workingSetVersions;

    private List<CategoryDTO> categories;

}
